projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -42,7 +43,9 @@ export class AuthService {
     protected oAuthLibWrapperService: OAuthLibWrapperService,
     protected authStorageService: AuthStorageService,
     protected authRedirectService: AuthRedirectService,
-    protected routingService: RoutingService
+    protected routingService: RoutingService,
+    @Optional()
```

Comment:
no need for `@Optional` decorator, because `AuthMultisiteIsolationService` is `providedIn: 'root'`